editor/src/components/editor/store/insertion-path.ts

```diff
@@ -68,6 +68,23 @@ export function childInsertionPath(
   }
 }
 
+export interface MapInsertionPath {
+  type: 'MAP_INSERTION'
+  intendedParentPath: StaticElementPath
+  insertBehavior: ConditionalClauseInsertBehavior
```

Comment:
This type should probably be renamed

Reply:
for the note, we agreed not to go with `MapInsertionPath` at all